History.txt

```diff
@@ -8,6 +8,7 @@ Changes between FreeRTOS 202104.00 and FreeRTOS 202107.00 released July 2021
 	+ Demo Updates
 		- Update Device Shadow demo to use Named Shadow feature.

```

Comment:
An entry for the Device Defender demo updates would be important

Reply:
Do you mean the json cleanup after removing parenthesis in the key macros? It's only a cosmetic change and does not include feature change.

Reply:
Okay, if it was only cosmetic, we don't need to mention about it.